tests/integration/flow.spec.ts

```diff
@@ -0,0 +1,40 @@
+import fs from 'fs';
+import path from 'path';
+import { executeCli } from '../helpers/execute';
+import { CLIENT_NAME, FILE_STORAGE_DIR, KID } from '../testConstants';
+
+const flowWorkWorkdir = path.join(FILE_STORAGE_DIR, 'flow');
+const flowPublicKeyPath = path.join(flowWorkWorkdir, 'publicKey.jwk');
+const flowPrivateKeyPath = path.join(flowWorkWorkdir, 'privateKey.jwk');
+
+jest.setTimeout(20000);
```

Comment:
wdyt about setting a timeout for each test instead of setting a global timeout in module level for jest?

Reply:
changed